common/species_rights/zz_hfe_vanilla_replace_species_rights.txt

```diff
@@ -57,21 +60,62 @@ citizenship_slavery = {
 		custom_tooltip = {
 			fail_text = does_not_embrace_species_slavery
 			# mod
-			or = {
-				has_species_flag = hfe_forever_slave
-				has_species_flag = hfe_forever_slave@from
-				has_trait = hfe_species_trait_happy_slave
-				from = {
+			OR = {
+				AND = {
+					OR = {
+						has_species_flag = hfe_forever_slave
+						has_species_flag = hfe_forever_slave@from
+					}
+					from = {
+						OR = { # The Ju Ahet will ALWAYS enslave Ahets
+							has_country_flag = hfe_egypt
+							has_country_flag = hfe_pirate
+							is_country_type = hfe_pirate
+							allows_slavery = yes
+						}
+					}
+				}
+				AND = {
+#					# For those who are not slavers, this Trait
+#					# just makes them docile
+					has_trait = hfe_species_trait_happy_slave
+					from = {
+						OR = { # The Ju Ahet will ALWAYS enslave Ahets
+							has_country_flag = hfe_egypt
+							has_country_flag = hfe_pirate
+							is_country_type = hfe_pirate
+							allows_slavery = yes
+							is_authoritarian = yes
+							is_xenophobe = yes
+#							NOT = {
+#								is_egalitarian = yes
+#								is_xenophile = yes
+#							}
+						}
+					}
+				}
+				# The Ju Ahet will ALWAYS enslave Ahets
+				from = { has_country_flag = hfe_egypt }
+				from = { has_country_flag = hfe_pirate }
+				from = { is_country_type = hfe_pirate }
+				from = { allows_slavery = yes }
+			}
+			from = {
+				OR = { 
 					allows_slavery = yes
-					# OR = {
-					# 	has_ethic = ethic_xenophobe
-					# 	has_ethic = ethic_fanatic_xenophobe
-					# 	has_ethic = ethic_authoritarian
-					# 	has_ethic = ethic_fanatic_authoritarian
-					# 	has_ethic = ethic_gestalt_consciousness
-					# }
+			 		has_ethic = ethic_xenophobe
```

Comment:
Look at allows_slavery definition

Reply:
Universal compatibility. Plenty of Mods, and Paradox themselves, define different types of slavery and Paradox regularly changes it.